### PR TITLE
Prepare new `constructor` stack release

### PR DESCRIPTION
**Pull request by jaimergp**
I've been building a new set of packages to `napari/label/bundle_tools_3`. It's almost ready so let's start the attempts. First a commit with no changes to get a control CI run.

### CODE REVIEW

**Comment by jaimergp** [comment]
Current CI pipeline failing due to a bug in conda/conda-libmamba-solver:

```
KeyError: "missing key 'https://conda.anaconda.org/napari/label/bundle_tools_2' in channel map: dict_keys([
'file:///Users/runner/work/packaging/packaging/conda-bld/osx-64',
'file:///Users/runner/work/packaging/packaging/conda-bld/noarch',
'https://conda.anaconda.org/napari/label/bundle_tools_2/osx-64',
'https://conda.anaconda.org/napari/label/bundle_tools_2/noarch',
'https://conda.anaconda.org/conda-forge/osx-64',
'https://conda.anaconda.org/conda-forge/noarch'
])"
```

(Note the absence of the subdir `osx-64` in the channel URL).

The packages in `napari/label/bundle_tools_2` are a bit old (last autumn builds) now so I have been revamping them in `bundle_tools_3`. It might need some polishing but they are more up-to-date. I'll use this PR to iterate and fix errors as we discover them during the week.

**Comment by jaimergp** [comment]
@Czaki - fixes are ready

File: .github/workflows/make_bundle_conda.yml [hunk 1]
```diff
@@ -4,10 +4,10 @@ on:
...
     branches:
       - main
-    paths:
-      - 'build_installers.py'
-      - 'conda-recipe/*'
-      - '.github/workflows/make_bundle_conda.yml'  # this file
+    # paths:
+    #   - 'build_installers.py'
+    #   - 'conda-recipe/*'
+    #   - '.github/workflows/make_bundle_conda.yml'  # this file
```
Comment (Czaki) [inline .github/workflows/make_bundle_conda.yml:10]: Why comment out this?

**Review by Czaki — APPROVED** [review]
In general looks ok

File: .github/workflows/make_bundle_conda.yml (hunk 1 above)
Comment (jaimergp) [reply .github/workflows/make_bundle_conda.yml:10]: Needed to trigger the workflow at first for a control check with `bundle_tools_2`. Thanks for the reminder!

Comment (jaimergp) [reply .github/workflows/make_bundle_conda.yml:10]:
```suggestion
    paths:
      - 'build_installers.py'
      - 'conda-recipe/*'
      - '.github/workflows/make_bundle_conda.yml'  # this file
```

**Pull request merged**